docker/cloudbuild.yaml

```diff
@@ -0,0 +1,52 @@
+options:
+  machineType: "N1_HIGHCPU_32"
+timeout: 1800s
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args: ['-c', 'docker login --username=lutter --password=$$PASSWORD']
```

Comment:
I'd make the username a variable too.

Reply:
This step isn't needed anymore anyway, since that all now happens in tag.sh. Removing this step.